lighthouse-core/runner.js

```diff
@@ -296,7 +296,7 @@ class Runner {
         if (noArtifact || noTrace) {
           log.warn('Runner',
```

Comment:
originally made this `log.verbose`, under the theory that you get a big red pageLoadError logged above this, but that's scrolled away and you just get a bunch of blue `status Auditing: Properly size images +1ms`, etc messages that make everything look ok. So now we just don't log here *and* below